registrations/admin.py

```diff
@@ -0,0 +1,58 @@
+from admin_auto_filters.filters import AutocompleteFilter
+from django.contrib import admin
+from django.utils.translation import ugettext as _
+from reversion.admin import VersionAdmin
+
+from registrations.models import Registration
+
+
+class BaseAdmin(admin.ModelAdmin):
```

Comment:
Are you planning to add more classes that use this baseclass? If it's just `RegistrationAdmin` then having this base class is a bit redundant.

Reply:
I think there is no need for it at the moment. I removed it. It can be added later if there is need for it